.github/ISSUE_TEMPLATE/1.bug_report.yaml

```diff
@@ -0,0 +1,227 @@
+          | ---------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------- |
+          | `com.amplifyframework:aws-analytics-pinpoint`        | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-analytics-pinpoint)             |
+          | `com.amplifyframework:aws-api`                       | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-api)                            |
+          | `com.amplifyframework:aws-api-appsync`               | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-api-appsync)                    |
+          | `com.amplifyframework:aws-auth-cognito`              | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-auth-cognito)                   |
+          | `com.amplifyframework:aws-datastore`                 | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-datastore)                      |
+          | `com.amplifyframework:aws-storage-s3`                | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-storage-s3)                     |
+          | `com.amplifyframework:aws-predictions`               | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-predictions)                    |
+          | `com.amplifyframework:aws-predictions-tensorflow`    | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/aws-predictions-tensorflow)         |
+          | `com.amplifyframework:core`                          | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/core)                               |
+          | `com.amplifyframework:core-kotlin`                   | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/core-kotlin)                        |
+          | `com.amplifyframework:rxbindings`                    | ![Maven Central](https://img.shields.io/maven-central/v/com.amplifyframework/rxbindings)                         |
+
+        </details>
+
+  - type: checkboxes
+    attributes:
+      label: |
+        Before opening, please confirm:
+      options:
+        - label: I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-android/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-android/discussions).
+          required: true
+        - label: I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-android/blob/release_v1.17.5/CONTRIBUTING.md#reporting-bugsfeature-requests).
+          required: true
+        - label: I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
+          required: true
```

Comment:
Yes, since we have provided input fields for the questions asked in bug reporting guide and some being required feilds, these questions seems irrelevant.